### PR TITLE
Add event requests bot

### PR DESCRIPTION
**Pull request by Dill-Lan**
Refactored copy of `/request_leave` for requesting to attend an event.

Added `/event-requests/slack-channel` to ssm

### CODE REVIEW

File: src/action_router.js [hunk 1]
```diff
@@ -11,31 +12,37 @@ exports.handler = async (event, context, callback) => {
...
-    if('dialog_submission' == payload.type){
-        if('leave_request_request' == payload.callback_id) {
-           responseBody = await LeaveRequests.handleSubmission(payload);
-        } 
-        else if('flight_request_request' == payload.callback_id) {
-           responseBody = await FlightRequests.handleSubmission(payload);
-        } 
-        else if('remote_request_casual_submission' == payload.callback_id) {
+    if ('dialog_submission' == payload.type) {
+        if ('leave_request_request' == payload.callback_id) {
```
Comment (seanarnold) [inline src/action_router.js]: Could maybe use a switch statement here

Comment (Dill-Lan) [reply src/action_router.js]: 👍

File: src/event_requests/actions.js [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+  else {
+    var response = { errors: [] };
+    if (!fromDate.isValid()) { response.errors.push({ name: 'from', error: 'Sorry, that date doesn\'t look right' }) }
+    if (!toDate.isValid()) { response.errors.push({ name: 'to', error: 'Sorry, that date doesn\'t look right' }) }
+    return response;
+  }
+}
+
+async function handleApproval(payload, action) {
+  console.log(payload, action);
```
Comment (jkRoberts) [inline src/event_requests/actions.js:59]: Should this be removed?

**Review by jkRoberts — APPROVED** [review]
Only the one in code comment about a console log, but otherwise, lgtm

**Pull request merged**